react/__tests__/components/Newsletter.test.js

```diff
@@ -89,4 +86,4 @@ test('should handle mutation error', async () => {
   const error = getByText(/something went wrong/i)
 
   expect(error).toBeTruthy()
-})
\ No newline at end of file
+})
```

Comment:
Nothing actually changed in this file, right?

Reply:
I changed because it was breaking on Travis. Make it use the function `waitForElement` instead of just `wait`